liblastlog2/src/lastlog2.h

```diff
@@ -25,9 +25,7 @@
   POSSIBILITY OF SUCH DAMAGE.
 */
 
-#pragma once
-
-#define _PATH_LASTLOG2 "/var/lib/lastlog/lastlog2.db"
+#define LL2_DEFAULT_DATABASE "/var/lib/lastlog/lastlog2.db"
```

Comment:
Did you think about letting callers just pass `NULL` to refer to the default location?
This avoids making the location part of the ABI.